Clinical-Genomics/patientMatcher
# Increase timeout for MongoDB connections

### This PR adds | fixes:
- Increase MongoDB parameter `serverSelectionTimeoutMS` to avoid timeout errors

### How to test:
Launch the demo server and make sure demo data is saved and it works as it should

### Review:

This [version](https://semver.org/) is a:
